openvdb_points/tools/AttributeSet.cc

```diff
@@ -99,7 +99,7 @@ AttributeSet::AttributeSet(const DescriptorPtr& descr, size_t arrayLength)
     for (Descriptor::ConstIterator it = mDescr->map().begin(),
         end = mDescr->map().end(); it != end; ++it) {
         const size_t pos = it->second;
-        mAttrs[pos] = AttributeArray::create(mDescr->type(pos), arrayLength);
+        mAttrs[pos] = AttributeArray::create(mDescr->type(pos), arrayLength, 1);
```

Comment:
The AttributeSet created from this constructor will not have the corresponding stride values or flags set from the AttributeSet the given descriptor was taken from. may be worth adding a comment for this in the header